src/main.css

```diff
@@ -0,0 +1,116 @@
+@import "/product_card-styles/stars.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+:root {
+  --product-card-width: 200px;
+  --product-card-border: 1px;
+  --product-card-padding: 16px;
+  --image-margin-bottom: 40px;
+  --title-margin-bottom: 4px;
+  --code-margin-bottom: 17px;
+  --rate-margin-bottom: 24px;
+  --each-star-margin-rigth: 4px;
+  --each-star-height: 16px;
+  --each-star-width: 16px;
+  --price-margin-bottom: 15px;
+  --color-for-purchase-block: #00acdc;
+}
+
+.product-card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: var(--product-card-width);
+  padding: var(--product-card-padding);
+  padding-top: 32px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  flex-direction: column;
+  color: #060b35;
+}
+
+.product-card__image {
+  display: flex;
+  box-sizing: border-box;
+  margin: 0 auto
+    var(--image-margin-bottom) auto;
+  padding: 0 3px;
+  width: 100%;
+  height: 134px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-top: 0;
+  margin-bottom: var(--title-margin-bottom);
+}
+
+.product-card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  margin-top: 0;
+  margin-bottom: var(--code-margin-bottom);
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 400;
```

Comment:
<img width="497" alt="Screenshot 2023-04-28 at 16 19 15" src="https://user-images.githubusercontent.com/53228259/235158344-70857731-babe-4f17-a0ad-f63873a2e3d2.png">
400 it is default value